packages/angular-logger/src/index.ts

```diff
@@ -0,0 +1,18 @@
+export { LoggerModule } from './lib/logger.module';
+
+export { Logger } from './lib/model/logger';
+export { ClassLogger } from './lib/model/class-logger';
+export { LogConsumer } from './lib/model/log-consumer';
+
+export {
+  setGlobalLogLevel,
+  createConsoleConsumer,
+  createNoopConsumer,
+  LoggerMark,
+  LogLevel,
+  globalLoggerInstance,
+  createClassLogger,
+  createLogger,
+  NamedLogger,
+  createNamedLogger,
+} from '@cloudflight/logger';
```

Comment:
why are we reexporting all those things?

Reply:
Because `@cloudflight/logger` is not a peer dependency.